code/game/objects/effects/anomalies/anomalies_bluespace.dm

```diff
@@ -66,3 +70,27 @@
 /obj/effect/anomaly/bluespace/proc/blue_effect(mob/make_sparkle)
 	make_sparkle.overlay_fullscreen("bluespace_flash", /atom/movable/screen/fullscreen/bluespace_sparkle, 1)
 	addtimer(CALLBACK(make_sparkle, TYPE_PROC_REF(/mob/, clear_fullscreen), "bluespace_flash"), 2 SECONDS)
+
+///Bigger, meaner, immortal bluespace anomaly
+/obj/effect/anomaly/bluespace/big
+	immortal = TRUE
+	teleport_range = 2
+	teleport_distance = 12
+	aSignal = null
+
+/obj/effect/anomaly/bluespace/big/Initialize(mapload, new_lifespan, drops_core)
+	. = ..()
+
+	transform *= 3
+
+/obj/effect/anomaly/bluespace/big/Bumped(atom/movable/bumpee)
+	if(iscarbon(bumpee))
+		var/mob/living/carbon/carbon = bumpee
+		carbon.reagents?.add_reagent(/datum/reagent/bluespace, 20)
+
+	if(isliving(bumpee))
+		var/mob/living/living = bumpee
+		living.apply_status_effect(/datum/status_effect/teleport_madness)
+	else
+		..()
```

Comment:
```suggestion
	if(!isliving(bumpee))
		return ..()
	var/mob/living/living = bumpee
	living.apply_status_effect(/datum/status_effect/teleport_madness)
```

Not the most essential early return but we like explicitly putting `return` before `..()` if it's the last thing you do as well.